src/core/stripe/schemas/checkout-params.schema.ts

```diff
@@ -8,15 +8,19 @@ export const LineItemSchema = z.object({
 
 // values from Stripe.Checkout.SessionCreateParams.Mode
 export const PaymentModesSchema = z.union([
-    z.literal("payment"),
-    z.literal("setup"),
-    z.literal("subscription"),
+    z.literal('payment'),
+    z.literal('setup'),
+    z.literal('subscription'),
 ]);
 
-
 export const CheckoutParamsSchema = z.object({
     customerId: z.optional(z.string()),
     email: z.optional(z.string().email()),
     lineItems: z.array(LineItemSchema).nonempty(),
     paymentMethod: PaymentModesSchema,
+    createCustomer: z.boolean(),
```

Comment:
Would it be possible to add a validation that assures that we are not sending at the same time the customerId and the createCustomer = true?
It would see missleading and Im not sure how stripe behaves if tell the API to use a customer id and create a new customer at the same time.